karabo_bridge/tests/test_client.py

```diff
@@ -18,6 +18,8 @@ def test_protocol_1(sim_server_version_1):
     data, metadata = c.next()
     assert 'SPB_DET_AGIPD1M-1/DET/0CH0:xtdf' in data
     assert 'SPB_DET_AGIPD1M-1/DET/0CH0:xtdf' in metadata
+    print(data['SPB_DET_AGIPD1M-1/DET/0CH0:xtdf'].keys())
+    print(metadata)
```

Comment:
Leftover debugging?

(I'm fine with leaving useful prints in the tests if you like, it can be helpful to have extra output when things go wrong)

Reply:
ah nice catch, yes they are leftovers :)